projekt_do_pracy_na_labach/fiszki/values.yaml

```diff
@@ -0,0 +1,86 @@
+# Default values for fiszki.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 3  # Ustawienie liczby replik
+
+image:
+  repository: nginx
```

Comment:
Jeżeli twój obraz do Fiszek nazywa się nginx, to ok, w innym przypadku polecałbym zmienić :p